pkg/plugins/policies/meshaccesslog/plugin/v1alpha1/access_log_v2.go

```diff
@@ -0,0 +1,147 @@
+package v1alpha1
```

Comment:
Why name the file v2? I think it's just a little confusing. Also @lobkovilya put this in another package `pkg/plugins/policies/meshtrafficpermission/xds/v3/rbac_configurer.go` I think it's a pretty good idea (probs without v3 though)

Reply:
Naming left over from when I had it in `listeners/v3`